locale/en_US/manager.po

```diff
@@ -146,6 +146,63 @@ msgstr "No announcement types have been created."
 msgid "manager.announcementTypes.typeName"
 msgstr "Announcement Type"
 
+msgid "manager.dois.actions.deposit.label"
+msgstr "Deposit DOIs"
+
+msgid "manager.dois.actions.deposit.prompt"
+msgstr "You are about to send DOI metadata records for {$count} item(s) to {$registrationAgency}. Are you sure you want to deposit these records?"
+
+msgid "manager.dois.actions.markRegistered.label"
+msgstr "Mark DOIs registered"
+
+msgid "manager.dois.actions.markRegistered.prompt"
+msgstr "You are about to mark DOI metadata records for {$count} item(s) as registered. Are you sure you want to mark these records as registered?"
+
+msgid "manager.dois.actions.export.label"
+msgstr "Export DOIs"
+
+msgid "manager.dois.actions.export.prompt"
+msgstr "You are about to export DOI metadata records for {$count} item(s) for {$registrationAgency}. Are you sure you want to export these records?"
+
+msgid "manager.dois.depositStatus"
+msgstr "Deposit Status"
+
+msgid "manager.dois.registration.submittedDescription"
+msgstr "The metadata for this item has been submitted to {$registrationAgency}"
+
+msgid "manager.dois.registration.notSubmittedDescription"
+msgstr "The metadata for this article has not been submitted to {$registrationAgency}"
```

Comment:
I prefer using "article" in this and the message above it. But that means they'll need to go to OJS.

Reply:
It just occurred to me that if we change it to article, we would need one for issue, and each of the types of pub objects for OMP and OPS. Would this be preferable from a user standpoint or would it be better to keep the number of more-or-less-the-same locale strings to a minimum?

Reply:
Good thinking, let's leave it at "item".